File: foofizzbuzzbar/foofizzbuzzbar.py
for B in range(1,1001):
	A=''
	if B%2==0:A+='Foo' 
	if B%3==0:A+='Fizz'
	if B%5==0:A+='Buzz'
	if B%7==0:A+='Bar'
	if A=='':A=str(B)
	print(A)

File: foofizzbuzzbar/foofizzbuzzbar.go
package main

import (
	"fmt"
	"strconv"
)
func main() {
	for i := 1; i < 1001; i++ {
		var s = ""
		if i%2 == 0 {s += "Foo"}
		if i%3 == 0 {s += "Fizz"}
		if i%5 == 0 {s += "Buzz"}
		if i%7 == 0 {s += "Bar"}
		if s == "" {s = strconv.Itoa(i)}
		fmt.Println(s)
	}
}
